contrib/dockerd-rootless.sh

```diff
@@ -77,6 +77,11 @@ if [ -z "$mtu" ]; then
 	mtu=1500
 fi
 
+dockerd="dockerd"
+if [ -n "$DOCKERD" ]; then
+	dockerd="$DOCKERD"
+fi
+
```

Comment:
This could be;

```suggestion
dockerd="${DOCKERD:-dockerd}"
```

Reply:
You are right, `command -v` works and is POSIX-compatible as well, making it better than the standalone `which`. I will commit both changes.